css-scroll-snap-2/Overview.bs

```diff
@@ -275,119 +275,246 @@ Snap Events {#snap-events}
+				<th>Event</th>
+				<th>Interface</th>
+				<th>Targets</th>
+				<th>Description</th>
+			</tr>
+		</thead>
 		<tbody>
 			<tr>
-				<th><dfn event>snapChanged</dfn>
-				<td>{{scroll!!event}}
+				<th><dfn for="snapchanged" event>snapchanged</dfn></th>
+				<td>{{SnapEvent}}</td>
+				<td>scroll containers</td>
+				<td>Fired at the scrolling element or {{Document}} at the end of a scroll (before a {{scrollend}} event)
+					or after a <a href="https://drafts.csswg.org/css-scroll-snap-1/#re-snap">layout snap</a>
+					if the element that the scrolling element or Document is snapped to changed.</td>
+			</tr>
 			<tr>
-				<th><dfn event>snapChanging</dfn>
-				<td>{{scroll!!event}}
+				<th><dfn for="snapchanging" event>snapchanging</dfn></th>
+				<td>{{SnapEvent}}</td>
+				<td>scroll containers</td>
+				<td>Fired at the scrolling element or {{Document}} during scrolling (before a {{scroll}} event),
+					if the element that the scrolling would cause the scroller to snap to is
+					different from the target reported by the last snapchanging event that was fired.</td>
+			</tr>
+		</tbody>
 	</table>
-	<h4>SnapEvents</h4>
-	<pre class="idl">
-			[Exposed=Window]
-			interface SnapEvent : Event {
-				constructor(DOMString type, optional SnapEventInit eventInitDict = {});
-				readonly attribute EventTarget? target;
-				readonly attribute SnapTargetList snappedTargets;
-				readonly attribute SnapTargetList snapTargets;
-				readonly attribute boolean invokedProgrammatically;
-				readonly attribute boolean smoothlyScrolled;
-			};
 
-			[Exposed=Window]
-			interface SnapTargetList {
-			    readonly attribute SnapTargetArray x;
-			    readonly attribute SnapTargetArray y;
+	<h4 for="snapchanged" id="snapchanged"> snapchanged </h4>
+	{{snapchanged}} indicates that the snap area to which a snap container is snapped along either axis has changed.
+	{{snapchanged}} should be dispatched:
+
+	<ol>
+	<li>
+		when a scrolling operation is <a spec="cssom-view-1" lt="scroll completed">completed</a>
+		if, for either the block or inline axis, the
+		element which the snap container is snapped to is different from the element
+		it most recently snapped to in that axis. For snap containers with
+		''scroll-snap-type/proximity'' strictness, the scroll may result in the snap
+		container no longer being snapped to any element.	[[css-scroll-snap-1#choosing]]
+		describes the method a UA follows when choosing	between elements which are
+		<a spec="css-scroll-snap-1" lt="scroll snap area">snap areas</a>.
+	</li>
+	<li> if there is a change to a snap container's style such that it goes from
+		having a non-'none' value for [[css-scroll-snap-1#scroll-snap-type|scroll-snap-type]]
+		to having a 'none' value or vice versa.
+	</li>
+	<li> if, after a [[css-scroll-snap-1#re-snap|layout change]], the element to
+		which a snap container is	snapped to changes, regardless of whether there is
+		a change in scroll position	after the layout change.
```

Comment:
nit: It seems like you've accidentally introduced some tabs in the middle of these sentences.

Reply:
Yes, deleted.

Reply:
There's still a tab on line 325

Reply:
Oops, fixed.